orttraining/orttraining/core/framework/checkpointing.cc

```diff
@@ -125,48 +52,77 @@ std::vector<std::string> GetOrderedOrtValueNames(const NameMLValMap& name_to_val
   return ordered_names;
 }
 
+}  // namespace
+
 Status SaveRuntimeTensors(
     const PathString& tensors_path,
     const PathString& tensors_data_path,
```

Comment:
Is this for external tensor data? If yes then can you change the name to something like tensor_external_data_path

Reply:
revert the change in checkpointing.h/cc as talked offline.